components/account/UserSets.vue

```diff
@@ -70,6 +83,9 @@
       },
       setPathMatch(set) {
         return set.id.replace('http://data.europeana.eu/set/', '');
+      },
+      capitalizeWord(word) {
+        return word.charAt(0).toUpperCase() + word.slice(1);
```

Comment:
Let's remove the need for this by changing the keys in the language file (again, sorry!) to `account.notifications.noCollections.public` and `account.notifications.noCollections.private`